roles/network_plugin/multus/defaults/main.yml

```diff
@@ -7,4 +7,5 @@ multus_cni_conf_dir: "{{ ('/host',  multus_cni_conf_dir_host) | join }}"
 multus_cni_bin_dir: "{{ ('/host', multus_cni_bin_dir_host) | join }}"
 multus_cni_run_dir: "{{ ('/host', multus_cni_run_dir_host) | join }}"
 multus_cni_version: "0.4.0"
+multus_crio_cni_version: "0.3.1"
```

Comment:
You can do this by overriding `multus_cni_version` in ansible group vars instead of doing this specialisation which breaks the role abstraction.

Reply:
I guess it’s not possible to do with an existing variable because this variable can be used only with applying from  worker nodes. But for crio nodes manifest should be applied from master node which can be with docker engine, for example. So, this will not help us.

Reply:
The problem is that manifest generates not per each k8s node in reality. You know what I mean ?

Reply:
this version string is an input for a jinja2 template not a kubernetes deployment file.